addons/message_fpn_trial_subscription_expiring/isFpnTrial.js

```diff
@@ -0,0 +1,8 @@
+(function(api, condition) {
+  //Enable message for users on the FPN 6 month trial plan only
+  const fpnPlanId = "price_1MzNRCJNcmPzuWtRMCwUWADu"
+  const subscriptionDataJson = JSON.parse(api.settings.subscriptionData)
+  const subscriptionPlanId = subscriptionDataJson["plan_id"]
+
+  subscriptionPlanId === fpnPlanId ? condition.enable() : condition.disable()
+});
```

Comment:
For future reference, a watcher should have been added to monitor `subscriptionData` changes so the condition could be re-evaluated when the subscription changes (eg signing out and back in, or signing into different account). Now, if a user switches from an account that shows this addon, to one that doesn't, they will still see it (and vice versa)

``` js
(function(api, condition) {
  function computeCondition() {

    //Enable message for users on the FPN 6 month trial plan only
    const fpnPlanId = "price_1J0Y1iKb9q6OnNsLXwdOFgDr"
    const subscriptionDataJson = JSON.parse(api.settings.subscriptionData)
    const subscriptionPlanId = subscriptionDataJson["plan_id"]

    subscriptionPlanId === fpnPlanId ? condition.enable() : condition.disable()
  }

  api.connectSignal(api.settings.subscriptionData, 'changed', () => computeCondition());

  computeCondition();
});
```